### ISSUE TITLE
Send Ack onReceipt of multiple read messages 

### ISSUE THREAD

**Issue by devilankur18** [tgalal/yowsup#1006]
I am receive a read receipt as follow

    DEBUG:yowsup.layers.logger.layer:rx:
    <receipt type="read" from="9194xxxxxxxx@s.whatsapp.net" id="1439905534-109" t="1439905618">
    <list>
    <item id="1439886750-93">
    </item>
    <item id="1439886917-94">
    </item>
    <item id="1439900181-95">
    </item>
    <item id="1439900349-96">
    </item>
    <item id="1439900939-98">
    </item>
    <item id="1439901223-99">
    </item>
    <item id="1439901484-100">
    </item>
    <item id="1439901894-101">
    </item>
    <item id="1439902060-102">
    </item>
    <item id="1439905321-106">
    </item>
    <item id="1439905322-107">
    </item>
    <item id="1439905470-108">
    </item>
    </list>
    </receipt>

I am using entity.ack() to generate a ack, but its sending it only for the main id 1439905534-109. I am keep getting the read receipt for the remaining items in the list in the same way.

**Comment by tgalal** [comment]
Acking the main id should indicate acking all items in contain and should work. So 2 questions: 

- are you sure the item Ids are always the same every time you receive them?
- are you sure you are sending the ack? (do you see it in logs going through?)

**Comment by devilankur18** [comment]
@tgalal Yeah main id was different and most of the items are on the list are same. I also tried sending the ack for all the items in the list, but also doesnot seems to solve the problem.



It seems more of some went wrong due to development usecase. Some time later, the list stop coming, not sure whats fixed it. I will keep and eye if this happens again.

**Comment by DiegoVallely** [comment]
@devilankur18  How do you doing to create the method for receiving ack of read? I guess I have the problem with this, can you tell me?

**Comment by jmmartinez84** [comment]
Same issue here:
`DEBUG:yowsup.layers.logger.layer:rx:`
`<receipt type="read" offline="0" from="XXXXXXXXXXX@s.whatsapp.net" id="1440441378-19" t="1441892935">`
`<list>`
`<item id="1440441382-23">`
`</item>`
`<item id="1440441381-21">`
`</item>`
`<item id="1440441377-18">`
`</item>`
`<item id="1441312923-119">`
`</item>`
`</list>`
`</receipt>`

`DEBUG:yowsup.layers.logger.layer:tx:`
`<ack to="XXXXXXXXXXX@s.whatsapp.net" type="read" id="1440441378-19" class="receipt">`
`</ack>`

`DEBUG:yowsup.layers.logger.layer:rx:`
`<receipt type="read" offline="0" from="XXXXXXXXXXX@s.whatsapp.net" id="1440441382-23" t="1441956312">`
`<list>`
`<item id="1440441381-21">`
`</item>`
`<item id="1440441377-18">`
`</item>`
`<item id="1441312923-119">`
`</item>`
`</list>`
`</receipt>`

`DEBUG:yowsup.layers.logger.layer:tx:`
`<ack to="XXXXXXXXXXX@s.whatsapp.net" type="read" id="1440441382-23" class="receipt">`

**Comment by subins2000** [comment]
Any updates on this ?
